plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1680,6 +1669,14 @@ fc::time_point producer_plugin_impl::calculate_block_deadline( const fc::time_po
    }
 }
 
+bool producer_plugin_impl::start_block_interrupted( const fc::time_point& deadline ) const {
```

Comment:
Not sure if the name `start_block_interrupted` can be changed. The method actually determines whether to interrupt `start_block` or not. But it sounds like `start_block` has already been interrupted. This causes a little confusion when you read the code below when `start_block_interrupted` is used.

Reply:
`is_start_block_interrupted` ?

Reply:
Or `can_interrupt_start_block` ? Hard to come up a good name.

Reply:
We use `should` in some places, how about `should_interrupt_start_block`.

Reply:
`should_interrupt_start_block` is good. Thanks.